source/threads.tex

```diff
@@ -7184,6 +7180,11 @@
 \end{note}
```

Comment:
What about this other note -- would it also work better outside the list?

Reply:
Maybe. I'm trying to limit the size of the changes this late in the process.

Reply:
We could further limit the size of the changes by turning these bullets into proper sentences. (There's already a period before the first note.) Eg:

> *Synchronization:* The following apply regardless of the provided `policy` argument:
> - The invocation of `async` synchronizes with the invocation of `f`.
>    [Note:...]
> - The completion of the function `f` is sequenced before the shared state is made ready.
>    [Note:...]

For master we could be a bit more aggressive:

> *Synchronization:* The invocation of `async` synchronizes with the invocation of `f`. The completion of the function `f` is sequenced before the shared state is made ready.
> [Note: These apply regardless of the provided `policy` argument, and even if the corresponding `future` object is moved to another thread. However, `f` might not be called at all, so its completion might never happen.]

Reply:
I like both parts of this.